ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/capella/BeaconBlockBodySchemaCapellaImpl.java

```diff
@@ -131,7 +131,7 @@ BlockBodyFields.EXECUTION_PAYLOAD, new ExecutionPayloadSchemaCapella(specConfig)
   @Override
   public SafeFuture<? extends BeaconBlockBody> createBlockBody(
       final Consumer<BeaconBlockBodyBuilder> builderConsumer) {
-    final BeaconBlockBodyBuilderCapella builder = new BeaconBlockBodyBuilderCapella().schema(this);
+    final BeaconBlockBodyBuilderCapella builder = new BeaconBlockBodyBuilderCapella(this, null);
```

Comment:
i think i'd prefer the previous version and having the second param initialise as null in the builder... really concerns me passing nulls across objects, pretty much ever... if it starts as null internally and gets set when required, then that'd be a better pattern.

Reply:
ye I was doing some back and forth on that. I landed here to avoid multiplication of constructors (up to 4 for each builder)

Reply:
My idea was not using the builder pattern for schema. But yeah.. I could go back on that decision...

Reply:
mmm the problem with set methods is that due to inheritance we will have all sets from supers with schema types for previous milestone.. it is bad. I think at the end I prefer the current one.